public/viewtopic.php

```diff
@@ -252,7 +252,7 @@
 
             if ($nextCommentAuthorised == 0) {
                 // Allow, only if this is MY comment (disclaimer: unofficial), or if I'm admin (disclaimer: unofficial, verify user?)
```

Comment:
This comment doesn't seem to align with the implementation, doesn't it?

Reply:
I'm not sure what the "disclaimer: unofficial" means, but the "MY comment" aligns with line 259, and the "I'm admin" aligns with line 255 (after my change).

Reply:
I'm guessing it's supposed to be "disclaimer: unverified" to indicate the disclaimer will be shown.

Reply:
Ah, yes - I see it now, my bad